docs/source/explanations/bundle_orientation.rst

```diff
@@ -0,0 +1,51 @@
+Bundle Orientation
+~~~~~~~~~~~~~~~~~~
+The result of tractometry is a tract profile, which describe how tissue
+properties vary along the length of a tract. These profiles are often discretized,
+and represented as 100 tissue property samples equally spaced along the length of
+the tract, from the start to the end. This raises the question of how tracts are
+oriented.
+
+In pyAFQ, the orientation is defined by the bundle definition. Tracts
```

Comment:
Can we link to somewhere where the technical term "bundle definition" is explained?